examples/fluids/README.md

```diff
@@ -26,6 +26,7 @@ Available runtime options are:
 | `-stab`                  | Stabilization method                                                                            |
 | `-implicit`              | Use implicit time integartor formulation                                                        |
 | `-bc_wall`               | Use wall boundary conditions on this list of faces                                              |
+| `-bc_outflow`            | Use outflow boundary conditions on this list of faces                                              |
```

Comment:
Should also include `-bc_inflow` and the new command line options to run with the different wind settings

Reply:
Well, I think I need to remove this but I need to add other command line options.